ls1intum/Artemis
# `Development`: Update Atlassian docker setup

### Motivation and Context



The bamboo version was updated in the gradle file, but not in the local docker setup. In this PR the local docker setup is updated to the Atlassian versions we use on our testing servers. 

### Description


### Steps for Testing

Execute the following commands to test the new version: 

```
cd src/main/docker
docker-compose -f atlassian.yml up -d
```
and verify that the Atlassian stack is running. 

Connect an Artemis instance to the docker setup and verify that programming exercises can be created again.




### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage






### Screenshots
